nixos/tests/kanidm.nix

```diff
@@ -44,6 +44,12 @@ import ./make-test-python.nix ({ pkgs, ... }:
         enableClient = true;
         clientSettings = {
           uri = "https://${serverDomain}";
+          verify_ca = true;
+          verify_hostnames = true;
+        };
+        enablePam = true;
+        unixSettings = {
+          pam_allowed_login_groups = [ "shell" ];
```

Comment:
Sorry to not have asked about this earlier, but why is this necessary? If I remove the line the test still succeeds.

Does the `shell` group have any special meaning on Unix?